services/class-wc-ebanx-checker.php

```diff
@@ -283,4 +283,19 @@ public static function check_https_protocol( $context ) {
 				->enqueue();
 		}
 	}
+
+	/**
+	 *
+	 * @throws Exception Param not found exception.
+	 */
+	public function validate_document() {
+		if (
+			WC_EBANX_Request::has( 'billing_country' )
```

Comment:
Can you apply a regex/pattern to validated this here ?
Which AR-document will have 11 characters ?
Can you create single test for this isolated method ?

Reply:
All AR-Documents must have 11 characters.